src/interface/compare.jl

```diff
@@ -55,3 +55,78 @@ end
 function Base.isequal(A::AbstractArray, B::AbstractTensor)
     return helper_isequal(A, B)
 end
+
+function helper_argmin(A, dims)
+    # @assert 1 <= max(dims) <= ndims(A) && 1 <= min(dims) <= ndims(A)
+    if (ndims(A) >= 2)
+        return map(x -> x[2], reduce(minby, map(Pair, A, CartesianIndices(size(A))), dims=dims, init=Inf=>CartesianIndex(fill(0, length(size(A)))...)))
+    else
+        return map(x -> x[2], reduce(minby, map(Pair, A, 1:size(A)[1]), dims=dims, init=Inf=>0))
+    end
+end
+
+function argmin_python(A; dims, keepdims)
```

Comment:
TODO: 
- [ ] Flattened
- [ ] Single Index always
- [ ] Look closer at the things returned in the reference

Reply:
https://data-apis.org/array-api/latest/API_specification/data_types.html#default-data-types

Reply:
Tuple should work with multi-dimensional indexing (is the default return type): https://data-apis.org/array-api/latest/API_specification/indexing.html#multi-axis-indexing